### PR TITLE
Add project existence

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

File: src/FileService.Broker/Requests/ProjectService.cs [hunk 1]
```diff
@@ -55,6 +59,18 @@ public async Task<List<ProjectUserData>> GetProjectUsersAsync(List<Guid> usersId
           _rcGetProjectUser,
           IGetProjectsUsersRequest.CreateObj(usersIds: usersIds),
           logger: _logger))?.Users;
-    }
+    }
+
+    public async Task<(ProjectStatusType projectStatus, ProjectUserRoleType? projectUserRole)> GetProjectUserRole(Guid projectId, Guid userId)
+    {
+      IGetProjectUserRoleResponse result = (await RequestHandler
```
Comment (MariyaSin) [inline src/FileService.Broker/Requests/ProjectService.cs:66]: could you return task?

File: src/FileService.Broker/Requests/ProjectService.cs [hunk 2]
```diff
@@ -55,6 +59,18 @@ public async Task<List<ProjectUserData>> GetProjectUsersAsync(List<Guid> usersId
...
+    {
+      IGetProjectUserRoleResponse result = (await RequestHandler
+        .ProcessRequest<IGetProjectUserRoleRequest, IGetProjectUserRoleResponse>(
+          _rcCheckProjectExistence,
+          IGetProjectUserRoleRequest.CreateObj(projectId: projectId, userId: userId),
+          logger: _logger));
+
+      return (result.ProjectStatus, result.ProjectUserRole);
+    }
+
```
Comment (MariyaSin) [inline src/FileService.Broker/Requests/ProjectService.cs]: remove empty line

**Review by MariyaSin — APPROVED** [review]

File: src/FileService.Broker/Requests/ProjectService.cs (hunk 2 above)
Comment (VLiamin) [reply src/FileService.Broker/Requests/ProjectService.cs]: done

File: src/FileService.Mappers/Models/ContentTypeMapper.cs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    private readonly FileExtensionContentTypeProvider _fileExtensionContentTypeProvider;
+
+    public ContentTypeMapper()
+    {
+      _fileExtensionContentTypeProvider =  new FileExtensionContentTypeProvider();
+    }
+
+    public string Map(string extension)
+    {
+      if (extension is null)
```
Comment (ndoriomedov) [inline src/FileService.Mappers/Models/ContentTypeMapper.cs:10]: can extension be empty? don't we need check in this case?

**Review by ndoriomedov — APPROVED** [review]

File: src/FileService.Mappers/Models/ContentTypeMapper.cs (hunk 3 above)
Comment (VLiamin) [reply src/FileService.Mappers/Models/ContentTypeMapper.cs:10]: Yes

**Pull request merged**